setup/dynamo_util.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  AttributeValue,
+  CreateTableCommand,
+  CreateTableCommandInput,
+  DynamoDBClient,
+  PutItemCommand,
+  PutItemCommandInput,
+} from "./deps.ts";
+
+export type CreateAndPut = {
+  table: CreateTableCommandInput;
+  items: Array<{
+    [key: string]: AttributeValue;
+  }>;
+};
+
+export async function createTableAndPutItems(
+  client: DynamoDBClient,
+  { table, items }: CreateAndPut,
+) {
+  await createTable(client, table);
+
+  // NOTE: Running `put` operations concurrently with `Promise.all` would lead to running out of write buffer.
+  for (const item of items) {
+    await put(client, {
+      TableName: table.TableName,
+      Item: item,
+    });
+  }
+}
+
+export function getCredFromEnv(): {
+  accessKeyId: string;
+  secretAccessKey: string;
+} {
+  const accessKeyId = Deno.env.get("AWS_ACCESS_KEY_ID");
+  const secretAccessKey = Deno.env.get("AWS_SECRET_ACCESS_KEY");
```

Comment:
I'm not really sure, but aws-sdk doesn't seem to fetch credentials from `~/.aws` in Deno. Seems like we have to use environment variables to specify them.